java-manta-client-unshaded/src/main/java/com/joyent/manta/client/crypto/ExternalSecurityProviderLoader.java

```diff
@@ -28,15 +36,26 @@
      */
     private static final Logger LOGGER = LoggerFactory.getLogger(ExternalSecurityProviderLoader.class);
 
+    /**
+     * System property identifier used to indicate the preferred security provider
+     * for client-side encryption.
+     */
+    private static final String PREFERRED_PROVIDERS_SYS_PROP_KEY = "manta.preferred.security.providers";
+
     /**
      * Name of BouncyCastle security provider.
      */
-    private static final String BC_PROVIDER_NAME = "org.bouncycastle.jce.provider.BouncyCastleProvider";
+    static final String BC_PROVIDER_NAME = "BC";
```

Comment:
Is this renaming just for convenience or laden with semantic meaning?

Reply:
The previous value was wrong. I'm going to reference the Bouncy Castle static variable for the name.